igm/modules/postproc/write_ncdf/write_ncdf.py

```diff
@@ -131,6 +131,11 @@ def update(params, state):
             E.axis = "X"
             E[:] = state.x.numpy()
 
+            try:
+                nc.pyproj_srs = state.pyproj_srs
+            except AttributeError:
+                nc.pyproj_srs = 'not provided'
```

Comment:
I think it would be cleaner to pass here - downstream you can ask `has_attr`, which is cleaner than fetching it and then try to convert it into a proj which will fail.